mscore/timeline.cpp

```diff
@@ -1264,6 +1264,7 @@ void Timeline::barlineMeta(Segment* seg, int* stagger, int pos)
                         // actually an end repeat followed by a start repeat, so nothing needs to be done here
                         break;
                   case BarLineType::DOUBLE:
+                  case BarLineType::DOUBLE_HEAVY:
```

Comment:
why not set add a special case for `DOUBLE_HEAVY`? Simply something like "Heavy double barline".

Reply:
'cause I've been lazy ;-)

Reply:
Brings up another issue: those strings are not translatable!

Reply:
(and also that I had forgotten to add a PixMap for that double heavy barline)